cms/djangoapps/contentstore/views/item.py

```diff
@@ -595,6 +595,18 @@ def _duplicate_item(parent_usage_key, duplicate_source_usage_key, user, display_
             else:
                 duplicate_metadata['display_name'] = _("Duplicate of '{0}'").format(source_item.display_name)
 
+        asides_to_create = []
+        for aside in source_item.runtime.get_asides(source_item):
+            for field in aside.fields.values():
+                if field.scope in (Scope.settings, Scope.content,) and field.is_set_on(aside):
+                    asides_to_create.append(aside)
+                    break
+
+        for aside in asides_to_create:
+            for field in aside.fields.values():
+                if field.scope not in (Scope.settings, Scope.content,):
+                    field.delete_from(aside)
```

Comment:
This works for now, but won't work as well if we ever get to the point where the data bindings for `Scope.content` and `Scope.settings` work the way that `XBlock` was designed, where edits are saved after the block leaves the context. If we get to that point, we'll want to actually create new asides, and then copy their attributes. But, for now, :+1: